client/src/modules/stock/entry/modals/lots.modal.html

```diff
@@ -75,15 +75,22 @@
       </div>
     </div>
 
-    <p ng-if="$ctrl.maxLotReached" class="alert alert-warning" style="margin: 5px 0px;">
-      <i class="fa fa-warning"></i> 
-      <span translate>FORM.INFO.MAX_LOT_REACHED</span>
-    </p>
+    <div ng-if="!$ctrl.isAllValid">
```

Comment:
Instead of having `!` in front of every `ngIf` here, it might be better to name compute these as values and name them appropriately.  For example, `!$ctrl.isAllValue` -> `$ctrl.hasInvalidEntries`, `!$ctrl.isLotExist` -> `$ctrl.hasNoLots`, `!$ctrl.isEveryLineCorrect` -> `$ctrl.hasIncorrectLines`, etc...

It makes it easier to read the HTML, since there isn't a `!` that's hiding in the expression.